core/scss/utilities/mixins/typography/_font-splash.scss

```diff
@@ -8,15 +8,10 @@
 // Style guide: Mixins.Typography.font-splash
 //
 @mixin font-splash {
-  @include h3;
+  @include modular-typography(4);
   @include margin(null null 0);
```

Comment:
```suggestion
  @include margin(null null 0);
  @include font-smoothing;
```

Reply:
Since this is a large font size, I'm adding font-smoothing we have consistent font rendering as the display fonts.